app/views/sites/api_instructions/_show_web_md.html.haml

```diff
@@ -1,485 +1,77 @@
 :markdown
-  <!-- Reminder to update https://open.gsa.gov/api/searchgov-results/ with any changes here. -->
-
-  This API exposes all relevant results “modules” in a single JSON call, including:
-
-
+  ## Overview
 
+  The Search.gov Results API exposes all relevant results “modules” in a single JSON call, including the following:
 
   * Web results;
   * Best bets;
   * Health topics;
   * Job openings;
   * Recent tweets;
-  * Recent news;
-  * Recent video news;
   * Federal Register documents; and
   * Related searches.
 
 
-  ## Note about Web Results and Endpoints
-
+  ## Getting started
 
+  Please refer to the detailed documentation for the Results API, now publicly available through the [OpenGSA Portal](https://open.gsa.gov/api/searchgov-results/). The information below serves as an introduction.
 
-
-  The endpoint you use to retrieve web results through this API will depend on the method we used to index your content. If we don't yet have your content indexed, you won't see results in the API.
-  
   We can index content using your [XML sitemap](https://search.gov/blog/sitemaps.html) (preferred), [RSS feeds](#{site_rss_feeds_path(@site)}), or the [indexing API, i14y](https://search.gov/developer/i14y.html). We can also deploy a crawler on a limited basis.
-  
-  Sites indexed via sitemaps, crawling, or i14y will use the `/search/i14y` endpoint. Sites indexed via RSS will use the top level `/search` endpoint.
-  
-  If we don't have your content indexed and you want to use the results API, you must [purchase an API key from Bing](#{commercial_keys_site_api_instructions_path(@site)}) to use their results. We are not able to pass the Bing results you may see on our hosted results page through APIs.
-
-  ## Resource URL
-
-
-
-
-  The endpoint is `#{api_scheme_and_host}/api/v2/search`.
-
-
-
-
-  You must use https.
-
-
 
+  Sites indexed via sitemaps or crawling will use the `/i14y` endpoint. Because most users are in this category, the example API calls in the documentation are to this endpoint. Sites indexed via RSS will use the top level `/` endpoint, please modify your calls accordingly.
 
-  ## Required Parameters
 
+  ## API Key and Resource URL
 
+  The endpoint is `#{api_scheme_and_host}/api/v2/search/i14y`.
+  `#{api_scheme_and_host}/api/v2/search/i14y?affiliate=#{h(@site.name)}&access_key=#{h(@site.api_access_key)}&query=SEARCH_TERM_ENTERED_IN_YOUR_SEARCH_BOX`
 
```

Comment:
Anybody using this _ought_ to know, but it wouldn't hurt to emphasize that the search term needs to be URL-encoded.